pkgs/ffigen/example/objective_c/avf_audio_bindings.dart

```diff
@@ -35168,25 +35768,46 @@ class NSString extends NSObject {
 
   void localizedStandardRangeOfString_(
       ffi.Pointer<_NSRange> stret, NSString str) {
-    _lib._objc_msgSend_340(
-        stret, _id, _lib._sel_localizedStandardRangeOfString_1, str._id);
+    _lib._objc_msgSend_useVariants1
```

Comment:
Are `@Native` annotations lazy loaded? If so this approach should work.

Reply:
Yes, they are "dynamic loading" everywhere currently.

If we'd ever to static linking, it would be after TFA. @sstrickl has been working passing the OS in to TFA, when I get to static linking, I could extend that work to cover `Abi`.